### PR TITLE
ITSASU-296: Update what tax year (individual)

### PR DESCRIPTION
**Pull request by llmikeyj**

### CODE REVIEW

File: app/views/individual/incometax/business/WhatYearToSignUp.scala.html [hunk 1]
```diff
@@ -38,7 +39,7 @@
 }
```
Comment (scullythornton1) [inline app/views/individual/incometax/business/WhatYearToSignUp.scala.html:39]: line 35 and 38 the question( heading) and title message for English (business.what_year_to_sign_up.heading.signup) will need to be updated to add For 

For which tax year do you want to start using software to file updates?

File: app/views/individual/incometax/business/WhatYearToSignUp.scala.html [hunk 2]
```diff
@@ -19,12 +19,13 @@
...
 @import views.html.helpers.injected.{ContinueButton, RadioHelper}
 @import views.html.templates.PrincipalMainTemplate
+@import scala.util.Properties
 
 @this(
-        mainTemplate: PrincipalMainTemplate,
-        form: FormWithCSRF,
-        radioHelper: RadioHelper,
-        continueButton: ContinueButton
+mainTemplate: PrincipalMainTemplate,
```
Comment (scullythornton1) [inline app/views/individual/incometax/business/WhatYearToSignUp.scala.html:25]: can you please indence line 25 to 28

File: app/views/individual/incometax/business/WhatYearToSignUp.scala.html [hunk 3]
```diff
@@ -71,16 +72,19 @@
...
+                    hint = Some(Hint(content = HtmlContent(optionCurrentContent))),
                     value = Some(Current.toString)
                 ),
                 RadioItem(
                     content = Text(messages("business.what_year_to_sign_up.option_2.signup", (endYearOfCurrentTaxPeriod).toString, (endYearOfCurrentTaxPeriod + 1).toString)),
-                    conditionalHtml = Some(optionNextContent),
+                    hint = Some(Hint(content = HtmlContent(optionNextContent))),
                     value = Some(Next.toString)
                 )
             )
```
Comment (scullythornton1) [inline app/views/individual/incometax/business/WhatYearToSignUp.scala.html:90]: can you please add one empty line after line 94

File: test/views/individual/incometax/business/WhatYearToSignUpViewSpec.scala [hunk 4]
```diff
@@ -60,12 +60,12 @@ class WhatYearToSignUpViewSpec extends ViewSpecTrait {
```
Comment (scullythornton1) [inline test/views/individual/incometax/business/WhatYearToSignUpViewSpec.scala:60]: can you please remove unused import line 26 and add an extra line in the end line 128.

File: app/views/helpers/injected/RadioHelper.scala.html [hunk 5]
```diff
@@ -13,9 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *@
+@import scala.util.Properties
 
 @this(
-        govukRadios: GovukRadios,
+govukRadios: GovukRadios,
```
Comment (scullythornton1) [inline app/views/helpers/injected/RadioHelper.scala.html:19]: can you indent this

File: app/views/helpers/injected/RadioHelper.scala.html [hunk 6]
```diff
@@ -39,7 +40,7 @@
         hint = hint.map { text =>
             Hint(
                 id = Some(s"${field.name}-hint"),
-                content = Text(text)
+                content = if (text.contains(Properties.lineSeparator)) HtmlContent("<p>" + text.replaceAll(Properties.lineSeparator, "<br>") + "</p>") else Text(text)
             )
         },
         items = radioItems,
```
Comment (scullythornton1) [inline app/views/helpers/injected/RadioHelper.scala.html:46]: add one extra line after line 49

File: test/assets/MessageLookup.scala [hunk 7]
```diff
@@ -540,17 +540,22 @@ object MessageLookup {
   }
 
   object WhatYearToSignUp {
-    val title = "For which tax year do you want to start using software to file updates?"
+    val title = "Which tax year do you want to start using software to file updates for?"
```
[the remark this answers is not in the document]
Comment (llmikeyj) [reply test/assets/MessageLookup.scala:543]: I believe this text is consistent with the spreadsheet mentioned in the related design ticket https://jira.tools.tax.service.gov.uk/browse/ITSASU-282

File: app/views/individual/incometax/business/WhatYearToSignUp.scala.html (hunk 2 above)
Comment (llmikeyj) [reply app/views/individual/incometax/business/WhatYearToSignUp.scala.html:25]: Certainly,

File: app/views/individual/incometax/business/WhatYearToSignUp.scala.html (hunk 3 above)
Comment (llmikeyj) [reply app/views/individual/incometax/business/WhatYearToSignUp.scala.html:90]: Ok,

File: test/views/individual/incometax/business/WhatYearToSignUpViewSpec.scala (hunk 4 above)
Comment (llmikeyj) [reply test/views/individual/incometax/business/WhatYearToSignUpViewSpec.scala:60]: Ok,

File: app/views/helpers/injected/RadioHelper.scala.html (hunk 5 above)
Comment (llmikeyj) [reply app/views/helpers/injected/RadioHelper.scala.html:19]: Ok,

File: app/views/helpers/injected/RadioHelper.scala.html (hunk 6 above)
Comment (llmikeyj) [reply app/views/helpers/injected/RadioHelper.scala.html:46]: Ok,

**Pull request merged**